src/Packages/Microsoft.Extensions.StaticAnalysis/README.md

```diff
@@ -1 +1,52 @@
-README
+# Microsoft.Extensions.StaticAnalysis
+
+A curated set of code analyzers and code analyzer settings.
+
+## Install the package
+
+From the command-line:
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.StaticAnalysis
+```
+
+Or directly in the C# project file:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Extensions.StaticAnalysis" Version="[CURRENTVERSION]" >
+    <PrivateAssets>all</PrivateAssets>
+    <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+  </PackageReference>
+</ItemGroup>
+```
+
+## Usage Example
+
+On install, a warning will be displayed that `The StaticAnalysisCodeType property is not defined, assuming 'General'`. The general ruleset is enabled by default. To select a different ruleset (or hide the warning) add the `StaticAnalysisCodeType` property to your project as follows.
```

Comment:
I think "ruleset" is the old word for this kind of stuff. In the Roslyn world these are called diagnostics. So the General set of diagnostics is enabled by default.